modules/util/source-formatter/src/main/resources/dependencies/replacements.json

```diff
@@ -16,6 +16,15 @@
 		"issueKey": "LPD-7751",
 		"to": "fetchCommerceCatalogByExternalReferenceCode(param#1#, param#0#)"
 	},
+	{
+		"from": "regex:serviceLocator.findService\\(\"com.liferay.dynamic.data.mapping.storage.StorageEngine\"\\)",
+		"issueKey": "LPD-7762",
+		"skipParametersValidation": true,
+		"to": "staticUtil[\"com.liferay.dynamic.data.mapping.kernel.StorageEngineManagerUtil\"]",
+		"validExtensions": [
+			"ftl"
+		]
```

Comment:
Hi @kyle-miho, can the service locator call be split into two lines?
Generally for good practice, we add optional \n in ".", "(", etc. To catch those cases.

Reply:
Or does the default logic for the regex in the code already deal with that?